node_modules/titanium-sdk/lib/emulators/genymotion.js

```diff
@@ -286,7 +286,10 @@ function findGenymotion(dir, config, callback) {
 	if (!player || !fs.existsSync(player)) {
 		player = path.join(dir, 'player' + exe);
 	}
-	if (player && !fs.existsSync(player)) {
+	if (!fs.existsSync(player) && process.platform != 'win32') {
```

Comment:
I [blame](https://github.com/FokkeZB/titanium_mobile/blame/patch-37/node_modules/titanium-sdk/lib/emulators/genymotion.js#L25) you for the `==` btw, thought I should have corrected it indeed ;)